indy_node/test/request_handlers/conftest.py

```diff
@@ -45,6 +51,32 @@ def schema_request():
                               }})
 
 
+@pytest.fixture(scope="module")
+def rs_encoding_handler(db_manager, write_auth_req_validator):
+    return RsEncodingHandler(db_manager, write_auth_req_validator)
+
+
+@pytest.fixture(scope="function")
+def rs_encoding_request():
```

Comment:
You already have had a function `build_get_rs_encoding_request`. I think it will be more clearly if that function will be used instead of copy-pasting. Also we have a helper function to transform txn to request and request to txn. They are placed in `plenum/common/txn_util.py`